apps/smart-contracts/core/test/unit/PrePOMarket.test.ts

```diff
@@ -705,20 +714,35 @@ describe('=> prePOMarket', () => {
       const shortToRedeem = amountMinted
       const totalOwed = await calculateTotalOwed(longToRedeem, shortToRedeem, false)
 
-      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem)
+      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem, user.address)
 
       expect(await longToken.balanceOf(user.address)).to.eq(amountMinted.sub(longToRedeem))
       expect(await shortToken.balanceOf(user.address)).to.eq(amountMinted.sub(shortToRedeem))
       expect(await collateralToken.balanceOf(user.address)).to.eq(totalOwed)
     })
 
+    it('correctly settles redemption with recipient other than funder before market end', async () => {
```

Comment:
```suggestion
    it('correctly settles redemption with recipient other than redeemer before market end', async () => {
```
Since we seem to be using redeemer nomenclature throughout this test, lets use it here as well.

Same for the other testcase you wrote.